### PR TITLE
[thelounge] Added new chart: The Lounge

### PR DESCRIPTION
**Pull request by rwaltr**
**Description of the change**

Added new Chart for [The Lounge](https://thelounge.chat/)

**Benefits**

New chart, new possibilities! 

**Possible drawbacks**

Yet another chart

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Review by DirtyCajunRice — APPROVED** [review]
Perfect!

**Comment by rwaltr** [comment]
CI running issues fixed
Ready for review

File: charts/stable/thelounge/values.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+image:
+  repository: thelounge/thelounge
+  pullPolicy: IfNotPresent
+  tag: 4.2.0-alpine
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the ${CHARTNAME} documentation
+# https://${CHARTNAME}.org/docs
```
Comment (bjw-s) [inline charts/stable/thelounge/values.yaml:17]: I think something went wrong here with the var substitution 🙂

**Review by bjw-s — CHANGES_REQUESTED** [review]

File: charts/stable/thelounge/values.yaml (hunk 1 above)
Comment (rwaltr) [reply charts/stable/thelounge/values.yaml:17]: That or I generated the file wrong.

Ill fix

**Review by bjw-s — APPROVED** [review]
Awesome, looks great!

**Comment by onedr0p** [comment]
@all-contributors add @rwaltr for code

**Pull request merged**